RELEASE_NOTE_DRAFT.md

```diff
@@ -1,27 +1,97 @@
+* Added Hindi Translation of Eclipse Collections website.
 
 # Bug Fixes
 -----------------
 * Fixed size edge case issues in Interval and IntInterval.
 
 # Tech Debt Reduction
 ---------------------
-* Optimized `removeIf` on UnifiedMap.
-* Implemented `removeIf` as a `default` method on MutableMapIterable.
-* Replaced usages of `Comparators.nullSafeEquals()` with `Objects.equals()`.
+* Replaced usage of org.eclipse.collections.impl.factory with org.eclipse.collections.api.factory wherever possible.
+* Added default method for RichIterable.aggregateBy() that takes a target Map.
+* Pulled up RichIterable.groupByUniqueKey() as default methods.
+* Pulled up implementations of aggregateBy() as default methods.
+* Pulled up implementations of with(), without(), withAll(), withoutAll() as default methods.
+* Pulled up OrderedIterable.toStack() as a default method.
+* Pulled up ListIterable.binarySearch() as default methods.
+* Simplified implementation of RichIterable.containsBy().
+* Added missing tests for MapIterable.getIfAbsent*().
+* Added missing tests for MultimapsTest.
+* Added missing tests for primitiveIterable.reduceIfEmpty().
+* Optimized ImmutableListFactoryImpl.toImmutable() by not creating a redundant array copy
+* Optimized Bag.aggregateBy() to use forEachWithOccurrences.
+* Optimized primitiveList.toImmutable() by not creating a redundant array copy.
+* Optimized collect methods for primitiveImmutableSingletonBag, primitiveImmutableSingletonSet, and primitiveImmutableSingletonList.
+* Optimized BooleanArrayList.removeIf().
+* Optimized IntInterval.size() and Interval.size() by caching size.
+* Optimized IntInterval.sum(), IntInterval.mean(), and IntInterval.average() by using direct formulas.
+* Removed duplicate code for implementations of aggregateBy().
+* Removed duplicate methods in AbstractMutableBagIterable and AbstractImmutableBagIterable that have been moved up as default methods.
+* Removed duplicate overrides for forEach().
+* Added Javadocs for primitivePrimitiveMaps, primitiveObjectMaps, objectPrimitiveMaps, primitiveValuesMaps and their hierarchy.
+* Improved documentation of Function2, Function3, and MutableCollection.injectIntoWith.
+* Enabled code generation of BooleanArrayStack from the common primitive stack template.
+* Added missing @Override annotations for tap().
+* Added and fixed Checkstyle checks for comma-separated lists that are partially wrapped.
+* Fixed Javadoc errors and warnings.
+* Fixed whitespace and line-wrapping violations.
+* Fixed inspections violations.
+* Fixed gradle dependency settings in website.
+
+# OSGi Integration Improvements
+-------------------------------
+* Added OSGi metadata.
+* Fixed OSGi metadata to have sun.misc be an optional import package.
+* Fixed project versions published to p2 repository.
 
 # Build Changes
 -----------------
-* Moved some maven configuration from .travis.yml to jvm.config.
-* Changed Maven plugin repository to use https.
+* Migrated to GitHub Actions.
```

Comment:
A suggestion to rephrase this line: 
Migrated all software development workflows or builds to GitHub Actions.

Reply:
fixed